content/swiftui/concepts/viewmodifier/terms/animation/animation.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: '.animation()'
+Description: 'Applies an animation to a view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.animation()`** modifier applies an animation to a [view](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+```pseudo
+< View to be animated >
+    < View modifier that varies depending on switchValue >
+    .animation(animation, value: switchValue)
+```
+
+The `.animation()` modifier takes two arguments:
+
+- An animation property or function, for example `.easeIn` or `.easeIn(duration: 2)`.
```

Comment:
I'd love to see a list of examples here. See: https://developer.apple.com/documentation/swiftui/animation/default

and additional type properties. This does in fact include `.easeIn`, `easeOut`, `linear`, `spring`, and much more. I recommend either creating a markdown table with the type property and a description column, or a simple bulleted list with the different (bullets like shown below):
 
- .typeProperty: Followed by a description.

Reply:
@yangc95 I added in a table or type properties and methods, thanks for the suggestion!